trio/_core/_tests/test_ki.py

```diff
@@ -488,21 +489,22 @@ async def main():
 
 
 # Regression test for #461
-def test_ki_with_broken_threads():
+# don't know if _active not being visible is a problem
```

Comment:
Probably more something to bring up with typeshed? But it's not documented AFAICT so probably intentional.

Reply:
Hmm, it's only `_active` and not `__active`, so I'm pretty sure a lot of code actually inspects it or modifies it in a subclass, but yeah makes sense that typeshed sticks to the documented parts.